pkg/sfu/downtrack.go

```diff
@@ -1478,3 +1494,32 @@ func (d *DownTrack) GetNackStats() (totalPackets uint32, totalRepeatedNACKs uint
 
 	return
 }
+
+func (d *DownTrack) onBindAndConnected() {
+	if d.connected.Load() && d.bound.Load() && d.kind == webrtc.RTPCodecTypeVideo && !d.bindAndConnectedOnce.Swap(true) {
+		targetLayers := d.forwarder.TargetLayers()
+		if targetLayers != InvalidLayers {
+			d.receiver.SendPLI(targetLayers.Spatial, true)
+		}
+
+		if d.activePaddingOnMuteUpTrack.Load() {
+			go d.sendPaddingOnMute()
+		}
+	}
+}
+
+func (d *DownTrack) sendPaddingOnMute() {
+	d.logger.Debugw("sending padding on mute")
+	// let uptrack have chance to send packet before we send padding
+	time.Sleep(waitBeforeSendPaddingOnMute)
+
+	for i := 0; i < maxPaddingOnMute; i++ {
+		if d.rtpStats.IsActive() || d.IsClosed() {
+			return
+		}
+
+		d.WritePaddingRTP(20, true)
```

Comment:
Is the sequence numbering working out okay? Forwarder has a `started` bool which gets set to `true` only in the non-padding path. I guess if there are padding packets sent before any real media, it is probably okay to send any sequence number, but just wanted to make sure.

Reply:
I think it would be ok too, the forwarder will update RTPMunger when real media packet received.